docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -2440,215 +2440,106 @@ And can be defined using different criteria like:
 
-.EXAMPLE: revent a device from connecting when its role is "default", the SSID is "SECURE", the current time is between 11am and 2pm, from Monday to Friday and as a registered device
-[source,ini,numbered]
-----
-  [category]
-  filter = node_info.category
-  operator = is
-  value = default
+.EXAMPLE_Reject_between_11am_2pm will revent a device from connecting when its role is "default", the SSID is "SECURE", the current time is between 11am and 2pm, from Monday to Friday and as a registered device
 
-  [ssid]
-  filter = ssid
-  operator = is
-  value = SECURE
+.EXAMPLE_Trigger_event_if_user will create a security event if the SSID is OPEN and the owner is igmout (the security event needs to have a custom trigger with the value 12345)
 
-  [time]
-  filter = time
-  operator = is
-  value = wd {Mon Tue Wed Thu Fri} hr {11am-2pm}
+.EXAMPLE_Autoregister_if_user will autoregister the device and assign the role staff to each device where the username is igmout.
 
-  [1:category&ssid&time]
-  scope = RegisteredRole
-  role = nointernet
-----
+.EXAMPLE_Autoregister_windows_devices will autoregister all Windows devices and assign them the role default.
 
-.EXAMPLE: create a security violaton if the SSID is OPEN and the owner is igmout
-[source,ini,numbered]
-----
-  [igmout]
-  filter = owner.pid
-  operator = is
-  value = igmout
+.EXAMPLE_Reject_specific_MAC will filter a MAC address and reject it by assigning the role REJECT.
 
-  [open]
-  filter = ssid
-  operator = is
-  value = OPEN
+.EXAMPLE_Detect_VOIP will auto set Avaya and Polycom as phones by matching vendor MAC, and set to default role
 
-  [2:igmout&ssid]
-  scope = RegisteredRole
-  action = trigger_security_event
-  action_param = mac = $mac, tid = SOMETHING_HAPPENED, type = CUSTOM
-----
+.EXAMPLE_Reject_User_Unless_Machine will refuse user authentication without prior machine authentication
 
-.EXAMPLE: autoregister the device and assign the role staff to each device where the username is igmout.
-[source,ini,numbered]
-----
-  [igmout]
-  filter = username
-  operator = is
-  value = igmout
+.EXAMPLE_Autoregister_Printer_Scanner will autoregister printers and scanners and add a note.
 
-  [secure]
-  filter = ssid
-  operator = is
-  value = SECURE
+You can have a look in the file `vlan_filters.conf`, there are some examples on how to use and define filters.
 
-  [3:igmout&secure]
-  scope = AutoRegister
-  role = staff
+Advanced LDAP Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  [4:igmout&secure]
-  scope = NodeInfoForAutoReg
-  role = staff
-----
+ldapfilter action
+^^^^^^^^^^^^^^^^^
 
-.EXAMPLE: autoregister all Windows devices and assign them the role default.
-[source,ini,numbered]
-----
-  [Windows_AutoReg]
-  filter = fingerbank_info.device_hierarchy_names
-  operator = includes
-  value = Windows
+ldapfilter action override the internal ldap filter packetfence create internally (uid=$username) so you can create a custom filter that match your needs.
```

Comment:
override -> overrides
ldap -> LDAP
packetfence create -> that PacketFence creates
that match -> that matches